docs/4.3. Controller to Authorization Server Interactions.md

```diff
@@ -108,6 +108,8 @@ The registration is done via a [dynamic client registration](https://specs.amwa.
 
 Depending on how the Authorization Server has been configured, it is likely that the Client Registration will need to be authenticated using an Initial Access Token. This token will be generated by the Authorization Server, and provided to the NMOS Controller by some proprietary method.
 
+When registering an NMOS Controller with more than one Authorization Server, the registration of unique `redirect_uris` for each Authorization Server can help in avoiding Authorization Server Mix-up attacks on your controller, whether it be a native application ([RFC8252, Section 8.10](https://tools.ietf.org/html/rfc8252#section-8.10)) or web based application ([OAuth 2.0 for Browser-Based Apps, Section 9.5](https://datatracker.ietf.org/doc/draft-ietf-oauth-browser-based-apps)).
```

Comment:
Suggestions:
* don't say `redirect_uris` as the plural term doesn't appear as far as I'm aware
* don't say "web based application" as RFC 6759 defines "web application" as something different than a "user-agent-based application" (a.k.a. browser-based apps) (cf. https://github.com/AMWA-TV/nmos-authorization/pull/107)
* style RFC with space before number for consistency with other references in NMOS
 
```suggestion
When registering an NMOS Controller with more than one Authorization Server, the registration of a unique `redirect_uri` for each Authorization Server can help in avoiding Authorization Server Mix-Up attacks on your controller, whether it be a native application ([RFC 8252, Section 8.10](https://tools.ietf.org/html/rfc8252#section-8.10)) or browser-based application ([OAuth 2.0 for Browser-Based Apps, Section 9.5](https://datatracker.ietf.org/doc/draft-ietf-oauth-browser-based-apps)).
```

Reply:
`redirect_uris` is the name of the parameter. There is no parameter called `rediect_uri`. Perhaps "...registration of a unique value for `redirect_uris` for each Auth..."

Reply:
Thanks. I'm a muppet, obviously! Fixed.